### ISSUE TITLE
Latest install deleted all saved messages older than 3 months without initiating any delete.

### ISSUE THREAD

**Issue by aaronwi** [forkgram/TelegramAndroid#187]
Latest install deleted all saved messages older than 3 months without user initiating any delete action.

**Comment by 23rd** [comment]
Did you check your saved messages from another Telegram clients?

**Comment by aaronwi** [comment]
> 



> Did you check your saved messages from another Telegram clients?

Yes, checked in nekox and desktop web client, all 4 years are gone

**Comment by 23rd** [comment]
So, how you came to your thought that Forkgram removed your messages?

**Comment by aaronwi** [comment]
Was the only recent change when it happened
